static/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -801,6 +802,11 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
       ? {...selection, datetime: {start, end, period: null, utc: null}}
       : selection;
 
+    const metricsWidgetFieldOptions = generateReleaseWidgetFieldOptions(
+      Object.keys(SESSIONS_FIELDS).map(key => SESSIONS_FIELDS[key]),
```

Comment:
Would this work here instead?
```suggestion
      Object.values(SESSIONS_FIELDS),
```